textx/metamodel.py

```diff
@@ -101,12 +101,14 @@ class TextXMetaModel(DebugPrinter):
             should interact), this attribute must be set via an optional
             constructor parameter "global_repository=True" or
             "global_repository=GlobalModelRepository()".
+        replace_regexp_with_groups (bool): if True, regexp terminals are
```

Comment:
Since this is in action when only one group is used I think singular name would be better `replace_regexp_with_group`.
Or just `use_regexp_group`.

Reply:
OK.